internal/logging/logger.go

```diff
@@ -13,72 +15,131 @@ type loggerKeyType int
 
 const loggerKey loggerKeyType = iota
 
+const (
+	messageKey = "message"
+	levelKey   = "level"
+)
+
 // NOTE - Logger is set to Nop as default to avoid redundant testing
 var logger *zap.Logger = zap.NewNop()
 
-// Config ... Configuration passed through to the logging constructor
-type Config struct {
-	UseCustom         bool
-	Level             int
-	DisableCaller     bool
-	DisableStacktrace bool
-	Encoding          string
-	OutputPaths       []string
-	ErrorOutputPaths  []string
```

Comment:
Any reason why we're removing configurability from logging? I don't think this change is warranted given that customizable encodings will be critical for other Pessimism operators to run the application on their own private infra